File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if(nums.length == 0){
            return 0;
        }
        int j = 0 ;
        for(int i = 1 ; i < nums.length ; i++){
            if(nums[i] != nums[j]){
                j++;
                nums[j] = nums[j + 1];
            }
        }
        return j + 1;
    }
}

File: Data Structures & Algorithms/move-zeroes/submission-0.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        result = []
        count = 0
        
        for i in nums:
            if i != 0:
                result.append(i)
            else:
                count += 1
                
        for i in range(count):
            result.append(0)
            
        nums[:] = result
